src/wallet/rpc/wallet.cpp

```diff
@@ -809,6 +809,61 @@ static RPCHelpMan migratewallet()
     };
 }
 
+RPCHelpMan getxpub()
```

Comment:
in 9900207360a2852041ceed260c3430e92ff860c5

The RPC can return xprv as well as xpub. We should find a better name as it becomes our API and users will depend on it.

How about `gethdkey`?

Reply:
Renamed as suggested